### ISSUE TITLE
Module not found: Can't resolve 'worker_threads'  - libsquoosh

### ISSUE THREAD

**Issue by yldoofYBRIK** [GoogleChromeLabs/squoosh#1079]
I am trying to use squoosh/lib in my react app but I am getting the following error 

```
Failed to compile.
./node_modules/@squoosh/lib/build/index.js
Module not found: Can't resolve 'worker_threads' in  'path/to/@squoosh/lib/build
```

has anyone else been running into this issue?

**Comment by surma** [comment]
What is your Node version?

**Comment by yldoofYBRIK** [comment]
14.17.1

**Comment by surma** [comment]
Just to be clear: libsquoosh doesn’t run in the browser (yet). You can’t use it with React. It’s a Node library.

**Comment by yldoofYBRIK** [comment]
I'm a little confused about how squoosh.app works in browser then if libsquoosh doesn't run in browser (yet).

**Comment by surma** [comment]
Yeah, I can see that. We wrote squoosh.app, then we wrote a Squoosh CLI. Both completely independent and separate (although they _do_ share the Wasm code). @atjn then extracted libsquoosh from the CLI, so that other build tools like Webpack, rollup etc could potentially make use of the image compression capabilities without having to shell out. 

Squoosh the PWA remains independent, tho. The main reason is that very different approaches are valid in Node but not valid on the Web (mostly related to file size and lazy loading).
